activerecord/lib/active_record/result.rb

```diff
@@ -239,6 +238,11 @@ def indexed_rows # :nodoc:
       end
     end
 
+    protected
+      def raw_column_types
```

Comment:
Leftover from a previous iteration?

Reply:
Yes, -_-.

Sorry, I haven't finished my coffee, should have taken the time to review my diff.

Reply:
Fixed in 7c7500f496d8a5d11d0b6e610e1b254ec34edf5a, thanks both for catching that.